src/Components/test/testassets/ComponentsApp.Server/Pages/Index.cshtml

```diff
@@ -9,7 +12,7 @@
     <link href="css/site.css" rel="stylesheet" />
 </head>
 <body>
-    <app>Loading...</app>
+    <app>@(await Html.RenderComponentAsync<App>())</app>
```

Comment:
This seems to be a bit redundant now. We now have both `<app>` and the rendering call.

Reply:
This is intentional for now. The `<app>` tag is the selector that replaces the prerendered content. I have ideas on how to remove that, but it's out of scope of this PR

Reply:
Both are needed. The `app` element gets replaced wholesale once the websocket connection is made - its a marker for the script. So it's included in a prerender, and then replaced,